fileutils.py

```diff
@@ -116,15 +116,15 @@ def __init__(self, fn):
         self.fn = fn
 
 
-def deprecate_old_json_files(config):  # TODO include !!
+def deprecate_old_output_files(config):  # TODO include !!
     """
     deletes older json files in savepath directory.
     experimental and loosely untested
     """
-    time_alive = int(config['auto_delete_old_json_files_after_x_minutes'])
+    time_alive = int(config['auto_delete_old_output_files_after_x_minutes'])
     user_selected_save_path = os.path.realpath(os.path.expandvars(config['savepath']))
     for f in os.listdir(user_selected_save_path):
-        if not f.endswith('json'):
+        if not f.endswith(('json', '.out')):
```

Comment:
very generic expression. could this start deleting other unrelated files? or is it safe due to dir?